constants.xml

```diff
@@ -392,6 +392,15 @@
             <wallet_address>ee06b3c906612cc5bdb087a30e6093c9f0aa04fd</wallet_address>
         </account>
     </ds_accounts>
+    <node_discovery>
+        <QUERY_DNS_FOR_SEED>true</QUERY_DNS_FOR_SEED>
+        <QUERY_DNS_MAX_TRIES>1</QUERY_DNS_MAX_TRIES>
```

Comment:
As the dns is not always accessible during my test, I added this in to retry in hope to access the dns.

But so far in the devop testnet, I don't see significant success rate with more retries.

Though I'm not quite sure would this be the same in mainnet, it could be good if we have this configurable to try out during the time when this is deployed to mainnet.